staging/src/k8s.io/api/networking/v1/generated.proto

```diff
@@ -444,31 +444,35 @@ message NetworkPolicyPort {
 
 // NetworkPolicySpec provides the specification of a NetworkPolicy
 message NetworkPolicySpec {
-  // Selects the pods to which this NetworkPolicy object applies. The array of
-  // ingress rules is applied to any pods selected by this field. Multiple network
-  // policies can select the same set of pods. In this case, the ingress rules for
-  // each are combined additively. This field is NOT optional and follows standard
+  // Selects the pods to which this NetworkPolicy object applies for ingress and/or egress
+  // (as determined by `policyTypes`).  The relevant one or ones of `ingress` and `egress`
+  // list connections that this policy allows to be made to/from the selected pods.
+  // Multiple NetworkPolicy objects can select a given pod, and the connections they
+  // allow are combined additively.
+  // If any NetworkPolicy applies to a given pod for ingress then (a) the default is
+  // to allow ingress to the pod only from the pod's node and (b) the applicable policies
+  // add allowed ingress to the pod; otherwise all ingress to the pod is allowed.
+  // If any NetworkPolicy applies to a given pod for egress then (a) the default is
+  // to deny all egress from the pod and (b) the applicable policies add allowed egress from the pod;
+  // otherwise all egress from the pod is allowed.
+  // A connection from some source pod to some destination pod is allowed if egress from the source
+  // is allowed and ingress to the destination is allowed.
```

Comment:
Maybe instead say “read the NetworkPolicy documentation to understand more about this API”?